### ISSUE TITLE
The hyperparameters of training resnet101 on Something-V1

### ISSUE THREAD

**Issue by withbrightmoon** [MCG-NJU/TDN#58]
Hi, thanks for the code! Could you tell me if the hyperparameters of training ResNet101 on Something-V1 are same to these of training ResNet50? that is,

`python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
            main.py  something  RGB --arch resnet50 --num_segments 8 --gd 20 --lr 0.01 \
            --lr_scheduler step --lr_steps  30 45 55 --epochs 60 --batch-size 8 \
            --wd 5e-4 --dropout 0.5 --consensus_type=avg --eval-freq=1 -j 4 --npb `
